src/RelayServer.ts

```diff
@@ -60,10 +60,10 @@ import Timeout = NodeJS.Timeout;
 const VERSION = '2.0.1';
 
 const calculateFeeValue = (
-    feePercentage: string,
+    feePercentage: string | undefined,
```

Comment:
Can we not do the following?:
```suggestion
    feePercentage?: string,
```

Reply:
and change the order of arguments in calculateFeeValue function

Reply:
updated, but will be pushed once I check it with Antonio :)